src/Parser.php

```diff
@@ -3419,6 +3430,9 @@ private function parseInterfaceDeclaration($parentNode) {
         return $interfaceDeclaration;
     }
 
+    /**
+     * @return InterfaceMembers
+     */
     private function parseInterfaceMembers($parentNode) : Node {
```

Comment:
Same, it's private so changing the real return type to InterfaceMembers should be safe